### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#9] [General]
# index

没什么有价值的内容的，真的！

https://lxl66566.github.io/

**Comment by dream-oyh** [comment]
啦啦啦！我来啦！评论系统能看到其他人的评论吗

**Reply by lxl66566** [reply]
可以的

**Comment by PerryKum** [comment]
啦啦啦啦啦 好东西喔

**Reply by lxl66566** [reply]
好耶

**Comment by ycsh-w** [comment]
我们都用了giscus，以及这个VuePress很像我在这里用的docusaurus：https://ecust-leap.github.io/

**Reply by lxl66566** [reply]
是的，它们主要只有基于 React 与 Vue 的区别；Ecust leap 是一个非常不错的项目。

**Reply by ycsh-w** [reply]
谢谢！希望你能多把我们的飞跃手册推荐给同学们 我们希望能帮助到尽可能多的同学
